kingfisher_scrapy/spiders/moldova_mtender.py

```diff
@@ -12,20 +12,20 @@ class MoldovaMTender(SimpleSpider):
     """
     name = 'moldova_mtender'
     data_type = 'release_package'
-    base_url = 'http://public.eprocurement.systems/ocds/tenders/'
 
     def start_requests(self):
         url = 'https://public.mtender.gov.md/tenders/'
         yield self.build_request(url, formatter=components(-1), callback=self.parse_list)
```

Comment:
Since `tenders` doesn't disambiguate requests, we can remove it.

```suggestion
        yield scrapy.Request(url, meta={'file_name': 'offset-0.json'}, callback=self.parse_list)
```

Reply:
I can't comment on the other line since it's not in the PR, but it would change:

```python
        yield self.build_request(url, formatter=join(components(-1), parameters('offset')), callback=self.parse_list)
```

to:

```python
        yield self.build_request(url, formatter=parameters('offset'), callback=self.parse_list)
```

Then we can remove the `join` import.